scout/server/utils.py

```diff
@@ -175,40 +175,6 @@ def user_institutes(store, login_user):
     return institutes
 
 
-def variant_case(store, case_obj, variant_obj):
-    """Pre-process case for the variant view.
-
-    Adds information about files from case obj to variant
-
-    Args:
-        store(scout.adapter.MongoAdapter)
-        case_obj(scout.models.Case)
-        variant_obj(scout.models.Variant)
-    """
-
-    chrom = None
-    starts = []
-    ends = []
-    for gene in variant_obj.get("genes", []):
-        common_info = gene.get("common")
-        if not common_info:
-            continue
-        chrom = common_info.get("chromosome")
-        starts.append(common_info.get("start"))
-        ends.append(common_info.get("end"))
-
-    if not (chrom and starts and ends):
-        return
-
-    try:
-        vcf_path = store.get_region_vcf(case_obj, chrom=chrom, start=min(starts), end=max(ends))
-
-        # Create a reduced VCF with variants in the region
```

Comment:
Was this used before and is no longer needed? Because if I grep `region_vcf_file` in the repo it doesn't look like it's used? 🤔

Reply:
No, I think we never used it for IGV.js, only for the old pileup.js-viewer. Sometimes it could be neat to have the variant in IGV as well for reference, but most of the time it is redundant.